libs/synapse/api-description/src/components/schemas/ChallengePagedResults.yaml

```diff
@@ -0,0 +1,23 @@
+title: ChallengePagedResults
+type: object
+properties:
+  totalNumberOfResults:
+    type: integer
+    description: 'The total number of results'
+  # results:
```

Comment:
I think this should exist as outlined here: https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/ChallengePagedResults.html

Reply:
I restored the property that I had commented out.